3001-4000/LIT3963Homily.xml

```diff
@@ -34,7 +35,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p/>
+            <p>Homily on <persName ref="PRS3899Eudoxia"/>, wife of Emperor <persName ref="PRS2016Arcadius"/> and on 
+               a certain Yoḥannǝs, who could be identified with <persName ref="PRS10390Yohanne"/> 
+               following <bibl><ptr target="bm:Lusini2005EAEDersanaM"/></bibl>, but according to <persName ref="DN"/>, is more 
+               likely to be identified with <persName ref="PRS5720JohnChr"/> due to the manuscripts' usual formulation 
+               <q xml:lang="gez">baʾǝnta Yoḥannǝs</q>.</p>
```

Comment:
add: "It is usually transmitted in the section for the month Gǝnbot."

Reply:
The month was already in the title but OK